grakn-graql/pom.xml

```diff
@@ -197,6 +197,11 @@
             <artifactId>commons-lang</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+        </dependency>
```

Comment:
@haikalpribadi I thought you re-added this, but pulling from you it was not there anymore, so I am re-adding it to make clear that we are depending on this dependency